### DISCUSSION TITLE
My network doesn't work. I have installed WIN 11 on MAC M1.

### DISCUSSION THREAD

**Discussion by BaimaCaio** [utmapp/UTM#5277] [Windows]
I install spice but i can't access internet yet.
I reinstall 1, 2, 3 times and the same.

Someone help-me

**Comment by BaimaCaio** [comment]
someone?

**Comment by ugurtasar** [comment]
Change your dns on windows and try again.

8.8.8.8
8.8.4.4

**Comment by vysecurity** [comment]
Doesn't work, its not a DNS issue. The network stack is broken.
